genie-agent/src/main/java/com/netflix/genie/agent/cli/AgentCLICommand.java

```diff
@@ -0,0 +1,30 @@
+/*
+ *
+ *  Copyright 2018 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+package com.netflix.genie.agent.cli;
+
+/**
+ * Interface for commands that the agent executes.
+ */
+public interface AgentCLICommand {
+
+    /**
+     * Entry point for command execution.
+     */
+    void run();
```

Comment:
Should this not return an integer or something so that eventually we can exit the application with that exit code?

Reply:
For the moment, I was going with either exit 0 or throw an exception, since FindBugs doesn't let me call `System.exit` anyway.
But I can take another shot at this and maybe see if findbugs is ok with it in certain cases (maybe only in `main`?)